missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/EndListener.java

```diff
@@ -68,21 +68,19 @@ public void onRespawn(PlayerRespawnEvent e) {
     }
 
     @EventHandler
-
     public void onDeath(PlayerDeathEvent e) {
-        if (!isInLobbyArea(e.getEntity().getLocation()))
-            return;
+        if (!isInLobbyArea(e.getEntity().getLocation())) return;
 
         Player p = e.getEntity();
-        p.setHealth(p.getMaxHealth());
+        p.setHealth(p.getAttribute(Attribute.GENERIC_MAX_HEALTH).getDefaultValue());
```

Comment:
I think that this is not compatible with 1.8 and some of the higher versions. Please stick to `p.setHealth(p.getMaxHealth());`